packages/docs/mintlify/compiler/fern-generate.mdx

```diff
@@ -8,6 +8,8 @@ You can run the fern compiler using the Fern CLI:
 fern generate --group <group name>
 ```
 
+_For local runs, you can use the `--local` option. More on this [here](/compiler/cli-reference.md#fern-generate---group-group---version-version)._
```

Comment:
```suggestion
To run the generators locally (rather than in the cloud), you can use the [`--local` option](/compiler/cli-reference.md#fern-generate---group-group---version-version).
```